app/assets/stylesheets/pages/_events_show.scss

```diff
@@ -0,0 +1,55 @@
+
+.logo {
```

Comment:
too general. Which logo? Are you sure nobody else will use a logo class anywhere in the app?

Reply:
this for the Logo Stravel since we only have one logo it shouldn't be an issue.

Reply:
ok